addons/hr_timesheet/models/project.py

```diff
@@ -301,6 +302,26 @@ def _compute_progress_hours(self):
                 task.progress = 0.0
                 task.overtime = 0
 
+    @api.depends('planned_hours', 'remaining_hours')
+    def _compute_remaining_hours_percentage(self):
+        for task in self:
+            if task.planned_hours > 0.0:
+                task.remaining_hours_percentage = task.remaining_hours / task.planned_hours
+            else:
+                task.remaining_hours_percentage = False
```

Comment:
Float fields default value is `0.0`. `False` would always be converted to that value anyway. See [here](https://github.com/odoo/odoo/blob/b9feebc25cc8caad97e5c3cb2953643bfd14984f/odoo/fields.py#L1495). It is better to ease the code reading and directly use `0.0`.
```suggestion
                task.remaining_hours_percentage = 0.0
```